### PR TITLE
Child theme cleanup / buildout

### PR DESCRIPTION
**Pull request by frrrances**
#### What does this PR do?
This PR finishes up a lot of Child Theme work that was left unfinished, like Category, Archive, and Search pages. 

#### Helpful background context (if appropriate)
Completing this work allows many of our deviant themes to be retired, like 150 books, Special collections, and fleshes out existing sites like AKDC. This work should _not affect_ the Exhibits site as that site uses custom posts. 

#### How can a reviewer manually see the effects of these changes?
This branch is deployed on test. You can view the Search, Category, and Archives pages for these sites: 
https://libraries-test.mit.edu/150books/
https://libraries-test.mit.edu/akdc/

This site also has the blog homepage and rich blog functionality:
https://libraries-test.mit.edu/digital-archives/

Other sites that will also get these improvements:
https://libraries-test.mit.edu/collections/
https://libraries-test.mit.edu/scholarly/

Should not be affected: 
https://libraries-test.mit.edu/exhibits/

#### What are the relevant tickets?
- https://mitlibraries.atlassian.net/browse/NTI-433
- https://mitlibraries.atlassian.net/browse/NTI-439

#### Todo:
- [x] UX review

#### Requires new or updated plugins, themes, or libraries?
NO

#### Requires change to deploy process?
NO

### CODE REVIEW

**Comment by frrrances** [comment]
@matt-bernhardt This is really ready for review now (or next week :) ). I believe I also fixed all the Travis issues in the files I touched. Let me know if you have any questions, and I'll ask UX to review next week too.

**Comment by frrrances** [comment]
@matt-bernhardt UX has given their sign off for this work to go live. Can I get a final review?

File: style.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Theme Name: MIT Libraries Child NEW
 Author: Lightning Trumpet & MIT Libraries
 Author URI: http://wordpress.org/
-Version: 2.2.1-@@branch-@@commit
+Version: 2.2.2-@@branch-@@commit
```
Comment (matt-bernhardt) [inline style.css:5]: Oh good, I was hoping that we bumped the theme version with this PR.

I'm not sure we have guidelines for when each number gets incremented - for this, I'm tempted to say we should make it 2.3.0, but there's time to have those conversations while we consider site redevelopment.

**Review by matt-bernhardt — APPROVED** [review]
This all looks good. I'm surprised that we're needing to explicitly include the archives page from the tag and category templates - but I gather that this is part of the inheritance problem that we've been grappling with.

:shipit:

**Pull request merged**